tests/helpers/parsers.js

```diff
@@ -66,12 +66,37 @@ const parsers = {
           testObject.parserOptions ? `parserOptions: ${JSON.stringify(testObject.parserOptions)}` : [],
           testObject.options ? `options: ${JSON.stringify(testObject.options)}` : []
         );
+
         const extraComment = `\n// ${extras.join(', ')}`;
+
+        // Augment expected fix code output with extraComment
+        const nextCode = { code: testObject.code + extraComment };
+        const nextOutput = testObject.output
+          ? { output: testObject.output + extraComment }
+          : undefined;
+
+        // Augment expected suggestion outputs with extraComment
+        const nextErrors = testObject.errors ? {
+          errors: testObject.errors.map((errorObject) => Object.assign(
+            {},
+            errorObject,
+            errorObject.suggestions ? {
+              suggestions: errorObject.suggestions.map(
+                (suggestion) => Object.assign(
+                  {},
+                  suggestion,
+                  { output: suggestion.output + extraComment }
+                )
+              ),
+            } : undefined)),
+        } : undefined;
```

Comment:
```suggestion
        const nextErrors = testObject.errors && {
          errors: testObject.errors.map((errorObject) => Object.assign(
            {},
            errorObject,
            errorObject.suggestions && {
              suggestions: errorObject.suggestions.map((suggestion) => Object.assign({}, suggestion, { output: suggestion.output + extraComment })),
            })),
        };
```

all those extra ternaries, and line length limits, are silly :-)